apps/sertra/components/ServerList.module.css

```diff
@@ -1,9 +1,15 @@
+.titleContainer,
+.tableHeader,
+.row {
+  width: 100%;
+  min-width: 40rem;
```

Comment:
It's to ensure that the name field retains a decent width. Since the mode/mod count/password columns at the end have fixed widths, on the narrow displays the name was quicky hidden, but I assume it's the most important bit of information and should remain visible at all times. I guess making the "extra columns" getting smaller/hidden would be the better solution now that I think of it? Maybe I should create another ticket for it?

Reply:
Gotcha, and I agree on narrow screens we should change it up. Right now for this phase one the designs are static in that regard, and we definitely need to address that, probably once we get this phase one out and to be tested in the first place.

I think the best way to go about it would be to lay the fields vertically for smaller screens. That would require adding the titles in the columns as well, hidden on large screens, of course, and consequently hiding the header row completely on smaller screens.

I'll have to reflect this on the designs, guess we'll cross that bridge when we get there.